pipeline/src/transformers/addressables.ts

```diff
@@ -28,7 +28,31 @@ import { transformAddressableExhibitionText } from './addressables/exhibitionTex
 import { transformAddressablePage } from './addressables/page';
 import { transformAddressableVisualStory } from './addressables/visualStory';
 
-export const transformAddressable = (
+export function transformAddressable(
```

Comment:
I rewrote this using [overload signatures](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads) so that the return type is known based on the input parameter type